ci/pingcap_chaos_mesh_build_kind.groovy

```diff
@@ -250,8 +257,11 @@ def call(BUILD_BRANCH, CREDENTIALS_ID) {
 		builds["E2E on kubernetes 1.12.10"] = {
                 build("v1.12", "${GLOBALS} GINKGO_NODES=6 KUBE_VERSION=v1.12.10 KIND_VERSION=0.8.1 ./hack/e2e.sh -- --ginkgo.focus='Basic'")
         }
-        builds["E2E on kubernetes 1.20.4"] = {
+        builds["E2E on kubernetes 1.20.2"] = {
                 build("v1.20", "${GLOBALS} GINKGO_NODES=6 KUBE_VERSION=v1.20.2 ./hack/e2e.sh -- --ginkgo.focus='Basic'")
+        }
+        builds["E2E on kubernetes 1.22.1"] = {
+                build("v1.22", "${GLOBALS} GINKGO_NODES=6 KUBE_VERSION=v1.22.1 ./hack/e2e.sh -- --ginkgo.focus='Basic'")
```

Comment:
For a long-term plan, we may need a document to announce the official support k8s version list of chaos mesh, and run e2e tests against these versions.

BTW, after we added this version, how many extra time cost has been introduced? From the [jenkins dashboard](https://ci.pingcap.net/blue/organizations/jenkins/chaos-mesh-e2e/activity), it looks like not a big problem 😄

Reply:
> For a long-term plan, we may need a document to announce the official support k8s version list of chaos mesh, and run e2e tests against these versions.

We are considering making a table about that, we are looking around for the similar things that other kubernetes operators do. :heart:

> BTW, after we added this version, how many extra time cost has been introduced? From the jenkins dashboard, it looks like not a big problem 😄

We execute each test in parallel, jenkins will create different workloads on different jenkins-worker(as different pods), so the time would not increase.